### ISSUE TITLE
WHAT is Rocket.Chat doing with scrollbars? (a11y and UI problem)

### ISSUE THREAD

**Issue by mirabilos** [RocketChat/Rocket.Chat#29359]
### Description:

Rocket.Chat, which I am forced to use for work, does weird things to scrollbars. They are not visible until I attach an external mouse with wheel and move the wheel (making scrolling impossible for Thinkpad classic X-series, trackball or 2/3-button mouse users, which is severely broken UI); they are then almost invisible (almost black on almost black; bad for accessibility, I can barely see it as is) and react much too sluggishly. They are also missing the up and down arrows.

I *do* have a Firefox `userContent.css` that fixes scrollbars on almost all other sites, such as this GitHub issue entry form, to behave somewhat properly (dark rectangle on brighter track, with arrows that work, always visible), but somehow Rocket.Chat manages to override this despite the `!important` in the `userContent.css` that’s supposed to prevent sites from doing just that.

### Steps to reproduce:

1. Log in to the Rocket.Chat webinterface
2. Try to find a channel, or go into a channel and try to read older messages

### Expected behavior:

The standard browser scrollbar is used and visible by standard browser rules. Users who configure an accessible scrollbar can use them in the web application.

### Actual behavior:

This… thing… seems to disable scrollbars, somehow, and draw its own. This is supposedly not possible, so it probably also overrides the `overflow` handling. Ugh. Major antifeature.

### Server Setup Information:

- Version of Rocket.Chat Server: I have no idea how to find this out
- Operating System: 
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### Client Setup Information

- Desktop App or Browser Version: Firefox 102.11.0esr
- Operating System: Debian stable

**Comment by Vaani-pathariya** [comment]
Willing to work on this issue , will raise a PR very soon

**Comment by jazztickets** [comment]
FWIW I've disabled "rc-scrollbars" by using this in Workspace -> Settings -> Layout -> Custom CSS
```
.rc-scrollbars-view { margin-right: 0px !important; }
.rc-scrollbars-track { display: none !important; }
```

**Comment by mirabilos** [comment]
jazztickets dixit:

>FWIW I've disabled "rc-scrollbars" by using this in Workspace -> Settings -> Layout -> Custom CSS
>```
>.rc-scrollbars-view { margin-right: 0px !important; }
>.rc-scrollbars-track { display: none !important; }
>```

OK, and how do I get the real scrollbars back?

**Comment by audouts** [comment]
Would fixing this help my issue at all?
https://github.com/RocketChat/Rocket.Chat/issues/31717

**Comment by mirabilos** [comment]
On Mon, 28 Oct 2024, audouts wrote:

>Would fixing this help my issue at all?
>https://github.com/RocketChat/Rocket.Chat/issues/31717

Possibly, but evidently rocketchat is actively disinterested in
fixing basic accessibility issues and honouring standards.

**Comment by audouts** [comment]
> On Mon, 28 Oct 2024, audouts wrote: Would fixing this help my issue at all? #31717
> Possibly, but evidently rocketchat is actively disinterested in fixing basic accessibility issues and honouring standards.

That's frustrating. Our team really likes RocketChat but it's frustrating that we have to hunt for tiny pixels to use an essential UI element. Frankly, the scrollbar is already pretty narrow.
